domains/mgb.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "belubruh123",
+           "email": "",
+           "discord": "911450938487750756",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.EGlOU0JmrXroUdIacWr5epMlsEF0Rt013gAbqVS_DMTeD3eVunT5Ly8iQC08fX0lz4r7u8gr1HkD46vdaRvP1fr1wxbeUTsOKpRyS7B6XnW4saN_TzYW3c05P1EyhXWcQ19Ad2zNrw_WCOP5nJ3CXr1pD3bUoSQXy26EiPWhIeCdu_kIBPeOPtxynWKP9DBKKy9OSAavTVyTgZDMZ4xuFMgdChPYp12o6y7fLl3GjoJoAz4QjgUYQjRD59-D-izVWt5e3alunxuDbTh2t9JQHaAlwGvk_9DnEmSEK1JeHSh2ypTBf9xl-PzV51A4bZi6UqT7d_laP3p92QFzGBe-_A.8KK6rHaDfUDlRWEQxL7zBg.NFZJyARnRzVgAwjpQ8vGTIv0YeTCbLgY90nZ4UKdXUMBOpcjZmaeaIxHOZKiEbA-r71oVpRUsgrBPVt5o4NsJHTr4YLSxL5fhghPqMxj-Y4.4G4GDDoCnc-7rduVxMWFeA"
+        },
+    
+        "record": {
+            "TXT": "google-site-verification=4gmncu4zpztqvl-v5w5t9bowmmoyuqq3-31kz_k0w-c"
```

Comment:
This should be under a google verification record and a google domain should be present.